exercises/ex_atm.py

```
@@ -0,0 +1,137 @@
+
+    Return:
+        The new changed dictionary of atm information.
+    """
+    try:
+        # ['check', 'withdrawal', 'deposit', 'change']:
+        if user_input == "check":
+            print(f"You'r balance: {atm_dict[user_id][BALANCE_INDEX]}")
+
+        elif user_input == "withdrawal":
+            withdrawal = input("Enter how much you want to withdrawal -> ")
+            if float(withdrawal) > 0:
+                atm_dict[user_id][BALANCE_INDEX] -= float(withdrawal)
+                print("You have withdrawn: " + withdrawal)
+
+        elif user_input == "deposit":
+            deposit = input("Enter how much you want to withdrawal -> ")
+            if float(deposit) > 0:
+                atm_dict[user_id][BALANCE_INDEX] += float(deposit)
+                print("Deposit succeeded")
+
+        elif user_input == "change":
+            new_password = input("Enter you'r new password -> ")
+            atm_dict[user_id][PASSWORD_INDEX] = new_password
+            print("Password changed to: " + new_password)
+
+        else:
+            print("Invalid input")
+
+    except ValueError:
+        print("Invalid input")
+
+    except Exception as e:
+        print(e)
+
+    finally:
+        return atm_dict
+
+
+def read_dict_move_to_file(file_path, atm_dict):
+    """Move the updated dictionary to the atm information file.
+
+    Args:
+        file_path (string): path to atm information file.
+        atm_dict (dictionary): the dictionary, containing all the atm
+                               information.
+    """
+    try:
+        with open(file_path, 'w') as file_pointer:
+            for key in atm_dict:
+                file_pointer.write(','.join([str(key), atm_dict[key][0],
+                                             str(atm_dict[key][1])]) + "\n")
+        print("Changes saved :)")
+
+    except FileNotFoundError:
+        print("Something went wrong, when accessing file")
```

The program is raising an exception saying the file was not found.
You catch it and instead print only partial information that something went wrong.
First thing would be to print all the information about the file the user tried to open and the fact that it doesn't exist.
Or you could simply just let the program crash throwing an exception saying that the file was not found.
Chose wisely ;)